src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,21 @@
 import React from 'react';
+import './CommentList.scss';
+import { Comment } from '../../react-app-env';
+import { CommentInfo } from '../CommentInfo';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+interface Props {
+  comments: Comment[] | undefined;
```

Comment:
if you pass an array, it cannot be `undefined`

```suggestion
  comments: Comment[];
```